azure-devops/run_build.yml

```diff
@@ -20,11 +20,11 @@ jobs:
       inputs:
         targetType: filePath
         filePath: $(Build.SourcesDirectory)/azure-devops/install_msvc_preview.ps1
-    - task: CacheBeta@0
+    - task: Cache@2
       displayName: Cache vcpkg
       timeoutInMinutes: 10
       inputs:
-        key: $(vcpkgResponseFile) | $(Build.SourcesDirectory)/.git/modules/vcpkg/HEAD

```

Comment:
the vcpkgResponseFile is not used anymore, the parameter definition at the top could be removed as well. But look to my other comment :)